### DISCUSSION TITLE
About test

### DISCUSSION THREAD

**Discussion by Darox360** [deepmodeling/deepmd-kit#1479] [Q&A]
Dear deepmd-kit developers,
    When I was testing, the program reported this error inexplicably, and no solution was found on the website. Please take a look at the error
![image](https://user-images.githubusercontent.com/80654702/154038084-62206f94-13de-4a61-ab07-07bb9cc72c28.png)

**Comment by AnguseZhang** [comment]
What's your model version and what's the software version? They should be the same.

**Comment by njzjz** [comment]
Looks like a same issue as #1062. If so, there's no way to fix it.
